### DISCUSSION TITLE
Form 提交时校验错误滚动区域支持到自定义DOM

### DISCUSSION THREAD

**Discussion by iambinlin** [ant-design/ant-design#35290] [Ideas]
<img width="1016" alt="image" src="https://user-images.githubusercontent.com/39019913/165665356-8b83e59f-01ca-4d81-aee4-04a10dd5312c.png">

假设我的表单是页面区域的一部分。表单域是没有滚动条的，也没有溢出，但是属于页面最下方，我触发页面级提交事件，触发整个页面的滚动～

**Comment by afc163** [comment]
目前支持的，可以在 https://ant.design/components/form-cn/#components-form-demo-register 测试效果。

**Reply by iambinlin** [reply]
> 目前支持的，可以在 https://ant.design/components/form-cn/#components-form-demo-register 测试效果。

我指的是：

```html
<div>
                <Header>
                    
                </Header>
                <Content></Content>
               <div>
                <Form>
                    
                </Form>
              </div>
                <Button>
                    
                </Button>
            </div>
```
这种形式的dom结构，我怎么触发最外层 div 的滚动呢

**Reply by iambinlin** [reply]
我写个 Demo吧

**Reply by iambinlin** [reply]
@afc163 https://codesandbox.io/s/zhu-ce-xin-yong-hu-antd-4-20-1-forked-vcm1h4?file=/demo.js

**Reply by iambinlin** [reply]
复现步骤，滚动到最底部，点击提交。

**Comment by iambinlin** [comment]
目前猜测的不生效的原因，可能是：

- 提交按钮没有内置进`Form`
- 用了`Affix`

**Reply by afc163** [reply]
```diff
  const handleSubmit = () => {
-   form.validateFields();
+   form.submit();
  };
```

**Reply by iambinlin** [reply]
可以了，感谢大佬！我想想，我应该配合 `onFinished` 事件，完成数据收集和接口调用。
